cmd/server/main.go

```diff
@@ -175,7 +175,14 @@ func main() {
 	customers.AddCustomerRoutes(logger, router, customerRepo, customerSSNStorage, ofac)
 	customers.AddCustomerAddressRoutes(logger, router, customerRepo)
 	documents.AddDisclaimerRoutes(logger, router, disclaimerRepo)
-	documents.AddDocumentRoutes(logger, router, documentRepo, bucket)
+
+	docsKeeper, err := secrets.OpenSecretKeeper(context.Background(), "customer-documents", os.Getenv("CLOUD_PROVIDER"))
```

Comment:
Can we use `DOCUMENTS_SECRET_PROVIDER` for this?

Reply:
No problem. 

PS, I shouldn't have created the PR yet. It's not actually ready for review. I'm improving the way I handle the multipart form to be a bit more defensive in case of abusively large payloads. After that I'll probably add more test coverage for documents.

Reply:
Made the change and documented it in configuration.md